gis/gdal/Portfile

```diff
@@ -36,6 +36,13 @@ use_xz              yes
 # error: call to 'abs' is ambiguous
 compiler.blacklist-append {clang < 900}
 
+# Enable use of 'macports-libcxx' for macOS 10.14 and earlier, as port uses
+# libcxx features normally only available on 10.15 and later.
+legacysupport.use_mp_libcxx \
+                    yes
+legacysupport.newest_darwin_requires_legacy \
+                    18
```

Comment:
If I'm not misstaken, the `legacysupport.use_mp_libcxx` is needed for 10.12 and earlier, then it should probably be:

```suggestion
# Enable use of 'macports-libcxx' for macOS 10.12 and earlier, as port uses
# libcxx features normally only available on 10.13 and later.
legacysupport.use_mp_libcxx \
                    yes
legacysupport.newest_darwin_requires_legacy \
                    16
```